src/hooks/useFormatNumber.mdx

```diff
@@ -1,18 +1,15 @@
 import { Meta, Source } from "@storybook/blocks";
+import ComponentDocsHeader from "../utils/components/ComponentDocsHeader";
 
 <Meta title="Hooks/useFormatNumber" />
 
-# useFormatNumber
-
-| Hook Version | DS Version |
-| ------------ | ---------- |
-| Added        | `3.5.3`    |
-| Latest       | `3.5.3`    |
-
-`useFormatNumber` provides a number with commas and handles a range of numbers.
-The formatting is based on standards from the NYPL Writing Style Guide.
-
-## Usage
+<ComponentDocsHeader
+  category="Reservoir Custom Hook"
+  componentName="useNumberFormat"
+  summary="Provides number formatting based on standards from the NYPL Writing Style Guide"
```

Comment:
I know we have a writing style guide, but I don't know where it is. We should link to it from storybook, unless if we are already and I missed it?